packages/tests-e2e/src/ui/b1-11/setup.ts

```diff
@@ -0,0 +1,14 @@
+import { test as setup } from '@playwright/test';
+
+import {
+  IdentifierEnforceVerification,
+  IdentifierType,
+  makeIdentifier,
+  setBackendConfigs,
+} from '../../utils/helperFunctions/setBackendConfigs';
+
+setup('set b1.11 configs', async (/* { page } */) => {
+  await setBackendConfigs('pro-8125835201872404154', [
```

Comment:
I would avoid repeating the projectID in multiple places => I would make this configurable because locally you have to use a different projectID than in the pipeline